sormas-backend/src/main/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjb.java

```diff
@@ -272,27 +267,41 @@ public long count(LabMessageCriteria criteria) {
 			labMessage.get(LabMessage.SAMPLE_OVERALL_TEST_RESULT),
 			labMessage.get(LabMessage.PERSON_FIRST_NAME),
 			labMessage.get(LabMessage.PERSON_LAST_NAME),
+			labMessage.get(LabMessage.PERSON_BIRTH_DATE_YYYY),
+			labMessage.get(LabMessage.PERSON_BIRTH_DATE_MM),
+			labMessage.get(LabMessage.PERSON_BIRTH_DATE_DD),
 			labMessage.get(LabMessage.PERSON_POSTAL_CODE),
 			labMessage.get(LabMessage.STATUS));
 
-		criteriaHandler(criteria, cb, cq, labMessage);
+		Predicate whereFilter = null;
+		if (criteria != null) {
+			whereFilter = labMessageService.buildCriteriaFilter(cb, labMessage, criteria);
+		}
+
+		// remove deleted entities from result
+		whereFilter = CriteriaBuilderHelper.and(cb, whereFilter, labMessageService.createDefaultFilter(cb, labMessage));
+		cq.where(whereFilter);
 
 		// Distinct is necessary here to avoid duplicate results due to the user role join in taskService.createAssigneeFilter
 		cq.distinct(true);
 
-		// remove deleted entities from result
-		Predicate filter = labMessageService.createDefaultFilter(cb, labMessage);
-		cq.where(filter);
```

Comment:
This second call to `cq.where` (first call in `criteriaHandler`) was the culprit for #7403.